src/types/intDecoding.ts

```diff
@@ -6,7 +6,7 @@ enum IntDecoding {
    * All integers will be decoded as Numbers, meaning any values greater than
    * Number.MAX_SAFE_INTEGER will lose precision.
    */
-  DEFAULT = 'default',
+  UNSAFE = 'unsafe',
```

Comment:
github won't let me comment in the right place.  Are we still using `MIXED`? I get the impression that you don't need it because you can use `bigint`, then your coerce routines will fix things up.

Reply:
I don't think we use currently using MIXED anywhere, but it's still a potentially useful option to have, especially since we now expose the JSON parsing to users

Reply:
I don't think I followed why we should expose the json parsing to users. Is the idea that they may not want our models, but want to use the sdk to hit an endpoint then parse it (differently) themselves?

Reply:
Right now it's expected that many of our type representations can be encoded into JSON and msgpack and back, and not just by our code -- we expect the user to be able to perform these encoding operations if they want to persist things to disk, for instance. Many of our types have a `get_object_for_encoding` method which returns an object ready to be serialized into one of those formats. Unfortunately bigints aren't supported by the standard JSON encoder, so without exposing the additional utilities, there wouldn't be an easy way to encode these newer objects in v3.

Technically this was also a problem in v2, but due to how types stored internal representations (they'd store exactly what they got most of the time), it was only a problem if you created a type with a bigint.